pe-parser-library/src/buffer.cpp

```diff
@@ -298,6 +298,28 @@ bounded_buffer *readFileToFileBuffer(const char *filePath) {
   return p;
 }
 
+bounded_buffer *makeBufferFromPointer(uint8_t *data, size_t sz) {
```

Comment:
Also: Since we `static_cast` to `uint32_t` for the `bounded_buffer` anyways, I think we might as well make this take a `std::uint32_t` instead of a `std::size_t`.